README.Rmd

```diff
@@ -102,4 +102,13 @@ plot_xctr(no_fin) +
   labs(title = "Risk distribution of all products on a portfolio level")
 ```
 
+### 4. Create a German map with risk categories color gradient
+
+```{r}
+german_map(financial, benchmark = "unit_isic_sec") +
+  labs(title = "German map of high, medium and low propotion of the companies that are found in one region.
+       © EuroGeographics for the administrative boundaries ")
```

Comment:
If there is a change that we'll support other countries, then I would also consider an API where the use prepares the data before passing it to a general `map_country()` function.

```r
financial |>
  prepare_germany() |>
  map_country()
```


In any case, the name `german_map()` seems to general. Maybe we need to find a name that more specifically reflect what kind of map this is. What if we later have another map of another kind?

Reply:
Great ! :) 
Since I know that I will have to plot other EU countries I have generalized the german_map() function to a map_region_risk() where the user can choose a country code which will be plotted.
The map_region_risk() function itself calls another internal function prepare_geo_data().
I think it will be better like that so that it will be more smooth to plot other countries.